subprojects/messaging/src/main/java/org/gradle/internal/serialize/ExceptionPlaceholder.java

```diff
@@ -60,7 +61,7 @@
     private Throwable toStringRuntimeExec;
     private Throwable getMessageExec;
 
-    public ExceptionPlaceholder(final Throwable throwable, Transformer<ExceptionReplacingObjectOutputStream, OutputStream> objectOutputStreamCreator) {
+    public ExceptionPlaceholder(Throwable throwable, Transformer<ExceptionReplacingObjectOutputStream, OutputStream> objectOutputStreamCreator) {
```

Comment:
My preference would be to use a local variable instead of reassigning the parameter. But that's more about style than any correctness.

Reply:
Point taken. I won't do this in this PR because paying the cost of going through CI just to change it isn't worth it.